frontera/contrib/middlewares/domain.py

```diff
@@ -87,10 +87,10 @@ def request_error(self, request, error):
         return self._add_domain(request)
 
     def _add_domain(self, obj):
-        obj.meta['domain'] = self.parse_domain_info(obj.url, self.manager.test_mode)
```

Comment:
It returns a dictionary with netloc, name, scheme, etc. The keys are bytes and the values are native strings. Do you think it would be better to convert the values too?